.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Great job @Jaymelfah 🤝

- [ ] You have done well in adding the `.github/workflows/linter.yml` file. As specified in the project requirements, you should also add the `.stylelintrc.json` and `.hintrc` files. This is the [correct file for the linters](https://github.com/microverseinc/linters-config/tree/master/html-css) for this project